Angle Values Calculation Added into TrackabilityManager.CollectMeasuredData()

mick-maclean opened the pull request:
This is part of my final year Computing Science Project as a BSc Data Science student at University of Stirling.

Sports department staff at the University find it frustrating angle values are not outputted alongside the timeseries data table when exporting to spreadsheet.
My patch fixes this with the following changes:

- MeasuredDataTimeseries.cs - I added the AngleValues variable to this class so as to store angle value data
- ExporterODS.cs - Additional code which inserts AngleValues and subheaders, if conditions are met.
- TrackabilityManager.cs - I have added code which, for a DrawingAngle with exactly three trackable points, retrieves its angle options and timeline data, builds filtered trajectories with calibration settings, computes the angular kinematics, and stores the resulting angular positions in mdt.AngleValues.

I have performed rigorous testing and found my angles have zero difference to the values created by existing Kinovea code shown in the graph within the Angular Kinematics tab (a screenshot of an example is attached at the bottom).

I have tried my best to edit as few files and lines of code as possible, I would not like this code to get in the way of other, more important matters in the overall Kinovea project. 

Thank you, Michael.

![Screenshot 2025-03-05 221223](https://github.com/user-attachments/assets/ee632815-33e5-40b4-b53a-9648d6513d46)

JoanCharmant reviewed (changes requested):
Thank you for doing this! I added a few comments related to code style to make it more homogeneous with the existing code. Other than that it looks good. 

One other thing I would like you to double check is if it still produce the same values when the user disables coordinates filtering, this is in Options > Preferences > Drawings > General > "Enable coordinates filtering". Normally it's already taken care of by FilteredTrajectory.Initialize() but it would be good to double check.

mick-maclean commented:
I have submitted a new commit with the corrected comments and line spacing, hopefully you can see that.

I have double checked for the filtered coordinates enabled/disabled and it look like FilteredTrajectory.Initialize() does take care of it already. I have added a screenshot below of the same video with filters turned off (left) and filters turned on (right) and added the data from Angular Kinematics tab like before.
![Screenshot 2025-03-09 111952](https://github.com/user-attachments/assets/52f6984a-c7cc-42e3-a6f6-9e0d600abb51)
